regcore/tests/db_django_models_tests.py

```diff
@@ -7,7 +7,11 @@
 from regcore.models import Diff, Layer, Notice, Regulation
 
 
-class ReusableDMRegulations(object):
+class DMRegulationsTest(TestCase):
+    def setUp(self):
+        Regulation.objects.all().delete()
```

Comment:
Django `TestCase` tests run within a transaction, right? Do we need these `delete` queries? The tests seem to pass without them.